src/NoteWorkspace.ts

```diff
@@ -1,6 +1,7 @@
 import * as vscode from 'vscode';
 import { basename, dirname, join, normalize, relative, resolve } from 'path';
 import { existsSync, readFile, writeFileSync } from 'fs';
+import { config } from 'process';
```

Comment:
I am going to merge but then delete this line. Seems like an unnecessary import.